### PR TITLE
[GITHUB-13] Allows broker id to be set via tag

### PR DESCRIPTION
**Pull request by lobsterdore**
Adds ability to specify broker id via a tag upfront, this
removes the need for the aws cluster autodiscovery script
when using this method.

The idea is to use a single ASG per instance, the ASG specifies
the broker.id via a tag which is picked up the instance, in
addition an EBS volume and ENI keep the instances data and
IP address constant (hence the addition of the interface
options).

Other changes:

* Upgrade to verson 0.10.1 by default
* Adds ability to specify an interface for bind and/or advertise
* Updates Kafka listener config to latest format

### CODE REVIEW

**Comment by lobsterdore** [comment]
Testing this at the moment.

**Comment by lobsterdore** [comment]
OK tested and looking good.

File: README.md [hunk 1]
```diff
@@ -45,6 +45,57 @@ To install run `ansible-galaxy install sansible.kafka` or add this to your
 
 and run `ansible-galaxy install -p ./roles -r roles.yml`
 
+### AWS Setup
```
Comment (quater) [inline README.md]: Line https://github.com/sansible/kafka/pull/14/files#diff-04c6e90faac2675aa89e2176d2eec7d8R43 should be updated to `version: v1.2`

**Comment by quater** [comment]
Apart from the tiny README.md update, LGTM 👍

**Comment by lobsterdore** [comment]
@quater readme fixed

**Comment by racer1680** [comment]
LGTM

**Pull request merged**